imap_processing/codice/constants.py

```diff
@@ -968,3 +972,287 @@
     254: 7864319,
     255: 4294967294,
 }
+
+OMNI_ENERGY_TABLE = {
```

Comment:
I am putting this here to be consistent with where I am storing other lookup tables. These values are not expected to change much, if at all.

Reply:
Are all of these contiguous regions with the min[1] == max[0]? If so, it might be easier to list all of the edges here and then calculate the centers/deltas from that one consistent list: `edges[1:] - edges[:-1]`

Reply:
Good idea, that will help reduce the amount of repeated values